packages/dashboard/src/components/SubwayNav/SubwayNav.tsx

```diff
@@ -0,0 +1,298 @@
+              key={'connector-sub' + '-' + stepToRender.key}
+            />
+          );
+
+          let subSteps = stepToRender.subSteps;
+          let prevStep: ISubwayNavStep | undefined = undefined;
+
+          if (subSteps !== undefined) {
+            subSteps.forEach((subStep: ISubwayNavStep, index: number) => {
+              subStepElements = subStepElements.concat(this._renderSubStep(stepToRender, subStep, index > 0 ? prevStep : undefined));
+              prevStep = subStep;
+            });
+
+            stepElements = stepElements.concat(subStepElements);
+          }
+        }
+      } // if stepToRender
+    } // if steps !== undefined
+
+    return stepElements;
+  }
+
+  /**
+   * Render substeps of current step in subway Nav control
+   * @param parentStep
+   * @param subStepToRender
+   * @param prevSubStep
+   */
+  private _renderSubStep(
+    parentStep: ISubwayNavStep | undefined,
+    subStepToRender: ISubwayNavStep | undefined,
+    prevSubStep: ISubwayNavStep | undefined
+  ): JSX.Element[] {
+    const stepElements: JSX.Element[] = [];
+    const { steps } = this.props;
+    let iconProps;
+
+    if (steps !== undefined) {
+      if (parentStep !== undefined) {
+        if (subStepToRender !== undefined) {
+          // Add the connectiong line
+          if (prevSubStep !== undefined) {
+            const connectorClass = this._getStepConnectorClassName(subStepToRender);
+            stepElements.push(
+              <div className={css(classNames.subwayNavSubStepConnector, connectorClass)} key={'connector' + '-' + subStepToRender.key} />
+            );
+          }
+
+          iconProps = this._getIconPropsSubStep(subStepToRender);
+
+          stepElements.push(
+            <div
+              key={subStepToRender.key}
+              className={css(
+                classNames.subwayNavStepDiv,
+                subStepToRender.state !== SubwayNavStepState.Current && subStepToRender.disabled ? classNames.disableStep : undefined
+              )}
+            >
+              <Icon iconName={iconProps.iconName} className={css(iconProps.iconClassName, classNames.subwayNavSubStepIcon)} />
+              <Link onClick={() => subStepToRender.onClickStep(parentStep, subStepToRender)}>
+                <span
+                  className={css(
+                    classNames.subStepLabel,
+                    subStepToRender.state === SubwayNavStepState.Current ? classNames.boldStep : undefined
+                  )}
+                >
+                  {subStepToRender.label}
+                </span>
+              </Link>
+            </div>
+          );
+        }
+      } // parentStep !== undefined
+    } // steps !== undefined
+    return stepElements;
+  }
+
+  /**
+   * Get the step connector class name
+   * @param step
+   */
+  private _getStepConnectorClassName(step: ISubwayNavStep | undefined): string {
```

Comment:
instead of using two different classes, maybe use two classes will result simpler code.

```
<div class="connector not-started" />
<div class="connector" />
```

Reply:
@brandonthomas  has some css changes.  I will look at this when integrating his changes

Reply:
This will be handled by the IStyleFunctionOrObject call I have